brouter-expressions/src/main/java/btools/expressions/BExpressionContext.java

```diff
@@ -857,6 +866,7 @@ int getVariableIdx(String name, boolean create) {
       if (create) {
         num = variableNumbers.size();
         variableNumbers.put(name, num);
+        lastAssignedExpression.add(null);
       } else {
```

Comment:
When you keep the parameter inserts, this needs a Null check like
if (lastAssignedExpression != null) lastAssignedExpression.add(null);